core/src/main/java/hudson/model/labels/LabelAutoCompleteSeeder.java

```diff
@@ -0,0 +1,61 @@
+package hudson.model.labels;
```

Comment:
I think we prefer to use `jenkins.*` packages for new classes right @oleg-nenashev / @daniel-beck ?

Reply:
I can move them, no problem.

Reply:
Yes, we should keep new API in "jenkins" if possible. Hudson is still a subject to potential trademark disputes